MTM-67166 Release notes for upcoming Spring boot 4 update in microservices SDK (#4961)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/cumulocity-undefined-microservices-sdk-spring-boot4-announcement.md

---
date: 
title: Upcoming Spring Boot version update for Microservice SDK  
change_type:
  - value: change-inv-3bw8e
    label: Announcement
product_area: Application enablement & solutions
component:
  - value: component-Sv2buFZ5l
    label: Microservice SDK
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
---

> We are pleased to announce the update to Spring Boot version 4.0.7 used in the Microservice SDK. This update requires updates to several third-party libraries and frameworks. These mandatory changes include:

Reviewer: ```suggestion
We are pleased to announce the update to Spring Boot version 4 used in the Microservice SDK. This update requires updates to several third-party libraries and frameworks. These mandatory changes include:
```

* Jetty 12.1.10

* Spring Security 7    

* Spring Framework: 7.0.8

**Impact**: Each of these updates may impact your microservices, potentially requiring code changes.

> The update is available in Continuous Deployment (CD) versions starting in Q3 2026 and in the annual releases 2026 and 2027.

Reviewer: ```suggestion
This update is now available in the Microservice SDK Continuous Deployment (CD) version X.Y.Z. We will share a separate announcement to confirm the 2026 release version that includes this change.
```

> The update is available in Continuous Deployment (CD) versions starting in Q3 2026 and in versions of the next annual release 2027.

Reviewer 2: also in the 2026 release

Author: adopted.

> **Migration Resources**: Refer to the following guides for assistance with the changes:

Reviewer 3: ```suggestion
**Migration resources**: Refer to the following guides for assistance with the changes:
```
